### ISSUE TITLE
Fundraising banner does not submit the correct donation amount

### ISSUE THREAD

**Issue by alexgibson** [mozilla/bedrock#8606]
## Description

When working on #8605 I noticed the fundarising banner on the [dev homepage](https://www-dev.allizom.org/en-US/) no passes the selected donation amount through to https://donate.mozilla.org/en-US/.

As a result, you need to select the amount again once you land on the foundation's page.

### Steps to reproduce

1. Visit https://www-dev.allizom.org/en-US/
2. Select "Monthly" and "$50".
3. Click "Donate"

### Expected result

When redirected to https://donate.mozilla.org/en-US/?frequency=monthly&amount=50&currency=usd&presets=50%2C30%2C20%2C10&ref=sept2019banner&utm_campaign=sept2019banner&utm_source=mozilla.org&utm_medium=referral&utm_content=banner

- The preselected amount should be $50 per month.

### Actual result

- No donation amount is selected.

### Environment

Firefox Nightly

**Comment by alexgibson** [comment]
Note that this bug doesn't currently cause any issues, as the banner is not in use. We should try and fix this before the next time the foundation ask to use it though ^

**Comment by TheoChevalier** [comment]
The code on bedrock is actually fine, this issue is we’re missing this feature on the new donate website. But it should be back soon!
This is tracked by https://github.com/mozilla/donate-wagtail/issues/844

cc @WillatMozFdn

**Comment by alexgibson** [comment]
@TheoChevalier this seems partly fixed (one time donations are passing through correctly), but I notice the monthly amounts don't seem right still. Do we need a follow-up issue to track that?

**Comment by TheoChevalier** [comment]
This should be fixed with https://github.com/mozilla/donate-wagtail/pull/969, which is merged, but pending a prod push

You can use the staging instance if you need to test different links https://donate-wagtail.mofostaging.net/en-US/?frequency=monthly&amount=50&currency=usd&presets=50%2C30%2C20%2C10&ref=sept2019banner&utm_campaign=sept2019banner&utm_source=mozilla.org&utm_medium=referral&utm_content=banner

**Comment by alexgibson** [comment]
Looks like this is now fixed in prod, thanks @TheoChevalier! 💯
